src/qt/clientmodel.h

```diff
@@ -54,6 +55,7 @@ class ClientModel : public QObject
     interfaces::Node& node() const { return m_node; }
     OptionsModel *getOptionsModel();
     PeerTableModel *getPeerTableModel();
+    PeerTableSortProxy* peerTableSortProxy();
```

Comment:
nit: could be noexcept, since it just returns pointer. Though since other similar getters here does not use `noexcept`, adding that one kinda seems out of scope.

Reply:
Though since other similar getters here does not use `noexcept`, adding that one kinda seems out of scope.

Going to leave it as is.

Reply:
FWIW I've suggested that this should not be "global" but rather instanced for each view (currently one in console dialog).